grafana/provider.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 var (
-	idRegexp  = regexp.MustCompile(`^\d+$`)
-	uidRegexp = regexp.MustCompile(`^[a-zA-Z0-9-_]+$`)
+	idRegexp    = regexp.MustCompile(`^\d+$`)
+	uidRegexp   = regexp.MustCompile(`^[a-zA-Z0-9-_]+$`)
+	emailRegexp = regexp.MustCompile(`.+\@.+\..+`)
```

Comment:
I'm going to let it pass, but validating email addresses with regular expressions is [impossible](https://www.regular-expressions.info/email.html) 100% of the time 😬

Reply:
Yeah, which is why I went super simple. (something)@(something).(something). I saw someone on hackernews say that person@tld was possible but very unlikely (who's using a TLD directly??)